### DISCUSSION TITLE
Why when creating unordered lists `-` (ul li elements) sometimes with p element is created but not all the time?

### DISCUSSION THREAD

**Discussion by rleyvasal** [quarto-dev/quarto-cli#5096] [Q&A] [answered]
What is the logic on markdown for rendering the `<p>` elements on unordered lists?  - important for styling elements with CSS

I have created a blog project  with markdown similar to the one below.  On the first bullet the `<p> `  html element is not created for the text in the bullet(ul li element), the second bullet creates a `<p> `  element, and the last nested bullet does not create `<p>`  again.

```markdown
---
title: "Post With Code"
author: ""
date: "2023-04-04"
categories: [news, code, analysis]
image: "image.jpg"
---

# Title 1.

The following bullet does not create a p html element. 

- This bullet does not create p element [quarto](https://https://quarto.org//)


# Title 2.

- *This bullet does create* a p element

  - Nested bullet **does not create** a  p element

````

The following screenshot shows the html for the first bullet 

![bullet_no_p](https://user-images.githubusercontent.com/61335136/229943457-85009763-5d7d-4d4f-86a9-841053e865e9.png)

The following screenshot shows the html for the second and third bullets

![bullet_p_nested_no_p](https://user-images.githubusercontent.com/61335136/229944673-b864546a-258f-47c9-8a35-6d4a194d1346.png)

**Comment by cscheid** [comment, marked as answer]
Markdown has a feature that distinguishes "tight" and "loose" lists:

https://stackoverflow.com/questions/43503528/extra-lines-appearing-between-list-items-in-github-markdown
